frontend/src/pages/projects/components/NotebookRestartAlert.tsx

```diff
@@ -0,0 +1,13 @@
+import * as React from 'react';
+import { Alert } from '@patternfly/react-core';
+
+const NotebookRestartAlert: React.FC = () => {
+  return (
+    <Alert variant="info" isInline title="Unsaved work will be lost">
+      Submitting changes causes associated workbenches to restart. To avoid losing your work, save
```

Comment:
Do we want to list the workbenches? cc @kywalker-rh

Reply:
- adding a storage directly to a workbench
- adding/removing a storage with a singular workbench either being added or removed from a list -- maybe both?
- adding/removing any number of data connections from workbenches
- directly editing a workbench

Aside from the last one & first one -- I think there is some chance of having multiple impacted. @DaoDaoNoCode do you think we can get the names listed?

Reply:
I think so, it should not be hard to implement.

Reply:
If it's only the one workbench restarting, I'll suggest some new microcopy:
"Updating this workbench will cause it to restart. To avoid losing your work, save any recent data."

^^This also matches the button copy (Update workbench)

LMK if we may need to list multiple workbenches and I can suggest a way to work that into the alert!

Reply:
@kaedward we definitely are looking at needing to list the names of workbenches. The UI doesn't hold onto everything. So you can remove a workbench and add a different workbench. Thus now it impacts two potentially running workbenches. 

Do we want to say "running x, not running y" or only if they are running? There is a lot of ways to go about this. "Impacted workbenches are: x, y, and z" (irrespective of running state).

Reply:
@andrewballantyne Will this still be an inline alert, and will it update as the user selects workbenches to update? And will the list of workbenches be included in that alert, or somewhere else? 

I think listing only the running workbenches should be fine, but I'm open to ideas. My reasoning is that the running ones are the workbenches that the user may want to check + save their work in before proceeding. I think if I got an alert like this I would want to make sure I checked ALL of the listed workbenches, running or not, especially if I skimmed the text and just saw "save your work". 

I'm imagining something like this:
"Running workbenches will restart upon updating. To avoid losing your work, save any recent data in the following running workbenches: X, Y, Z"